packages/esm-patient-chart-app/src/clinical-views/hooks/useFormsJson.ts

```diff
@@ -4,7 +4,7 @@ import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
 import { type Form } from '../types';
 
 export function useFormsJson(formUuid: string) {
-  const url = `${restBaseUrl}/form/${formUuid}`;
+  const url = formUuid ? `${restBaseUrl}/form/${formUuid}` : null;
   const { data, isLoading, error } = useSWR<{ data: Form }, Error>(url, openmrsFetch);
```

Comment:
nit: Maybe setting the URL conditionally is a better pattern here:

```suggestion
  const { data, isLoading, error } = useSWR<{ data: Form }, Error>(formUuid ? url : null, openmrsFetch);
```